polkadot/xcm/xcm-runtime-apis/tests/mock.rs

```diff
@@ -349,7 +348,7 @@ impl pallet_xcm::Config for TestRuntime {
 	type RuntimeOrigin = RuntimeOrigin;
 	type RuntimeCall = RuntimeCall;
 	const VERSION_DISCOVERY_QUEUE_SIZE: u32 = 100;
-	type AdvertisedXcmVersion = AdvertisedXcmVersion;
+	type AdvertisedXcmVersion = pallet_xcm::CurrentXcmVersion;
```

Comment:
Is there any issue in advertising the latest XCM version?

Reply:
no, not at all, `pallet_xcm::CurrentXcmVersion;` is the latest and works, 
I just wanted to align `type AdvertisedXcmVersion` here, because totally everywhere we use: `type AdvertisedXcmVersion = pallet_xcm::CurrentXcmVersion;`.

Also I have a follow-up [issue](https://github.com/paritytech/polkadot-sdk/issues/3214) for this XCM version stuff e.g.  `AdvertisedXcmVersion` / `CurrentXcmVersion` / `XCM_VERSION` / `SafeXcmVersion`